nntrainer/utils/node_exporter.cpp

```diff
@@ -108,12 +108,14 @@ void Exporter::saveTflResult(const std::tuple<props::Unit> &props,
   auto weight_transform = [](std::vector<const Tensor *> &weights) {
     std::vector<Tensor> new_weights;
     new_weights.reserve(weights.size());
-    new_weights.push_back(weights[0]->transpose("0:2:1"));
+
+    new_weights.push_back(weights[0]->transpose("0:2:1", true));
```

Comment:
We just do the operation properly here. and update new_weights. No need to use transpose function here.

Reply:
Since all other layers have been transposed, I think FC layer should also be transposed. Is it unnecessary?